mmpose/apis/inference.py

```diff
@@ -492,11 +495,19 @@ def inference_bottom_up_pose_model(model,
 
         returned_outputs.append(h.layer_outputs)
 
-        for pred in result['preds']:
+        for idx, pred in enumerate(result['preds']):
+            area = (np.max(pred[:, 0]) - np.min(pred[:, 0])) * (
+                np.max(pred[:, 1]) - np.min(pred[:, 1]))
             pose_results.append({
                 'keypoints': pred[:, :3],
+                'score': result['scores'][idx],
+                'area': area,
             })
 
+        # pose nms
+        keep = oks_nms(list(pose_results), oks_thr, sigmas=None)
```

Comment:
pose_results should be list already?

Reply:
It seems that `list` is not necessary here.

Reply:
I have deleted `list` in inference and (bottom up & top down) dataset files.